pkg/controller/operconfig/operconfig_controller.go

```diff
@@ -320,6 +322,14 @@ func (r *ReconcileOperConfig) Reconcile(ctx context.Context, request reconcile.R
 				return reconcile.Result{}, err
 			}
 		}
+		anno := obj.GetAnnotations()
```

Comment:
I think I'd rather this live in `pkg/apply`, since the need to copy objects may not be limited to the operconfig controller.

Reply:
Accordingly, you'll have to pass the full set of Clients to all the controllers. But that was inevitable anyways.